src/Raven.Client/Documents/Operations/AI/Agents/AiAgentConfiguration.cs

```diff
@@ -26,11 +35,42 @@ public AiAgentConfiguration(string connectionStringName, string systemPrompt)
         SystemPrompt = systemPrompt;
     }
 
+    /// <summary>
+    /// The name of the connection string used to connect to the AI provider.
+    /// </summary>
     public string ConnectionStringName { get; set; }
+
+    /// <summary>
+    /// The prompt that guides the behavior and purpose of the AI agent.
+    /// </summary>
     public string SystemPrompt { get; set; }
+
+    /// <summary>
+    /// A JSON schema (as string) describing the expected structure of the AI agent's output.
+    /// This allows validation and parsing of the AI-generated response according to a known format.
+    ///
+    /// For example:
+    /// <code>
+    /// {
+    ///   "Answer": "Answer to the user question",
+    ///   "Relevant": true,
+    ///   "RelevantOrdersId": ["The order ids relevant to the query or response"],
+    ///   "MatchingProductsId": ["All the product ids referenced either by the user or the system"]
+    /// }
+    /// </code>
+    /// </summary>
```

Comment:
We should not use this - this is a `(sampleObject / Schema)` thing - and hopefully not using strings like that

Reply:
We don't use it, it is generated from the schema type

Reply:
We do, here it is. Or am I missing something?